maplibre/src/render/resource/surface.rs

```diff
@@ -1,9 +1,10 @@
 //! Utilities for handling surfaces which can be either headless or headed. A headed surface has
 //! a handle to a window. A headless surface renders to a texture.
 
-use std::{mem::size_of, num::NonZeroU32, sync::Arc};
-
 use log::debug;
+use std::{mem::size_of, num::NonZeroU32, sync::Arc};
+#[cfg(feature = "headless")]
```

Comment:
minor remark: I usually just write out the paths where it is used instead of using/import.

Reply:
:+1:  thank you for pushing the fix ;)